src/main/resources/templates/pdf/legalStatementProbate.html

```diff
@@ -44,6 +44,7 @@
 </head>
 <body>
 <table>
+    {% set numberOfCodicils = case_details.case_data.codicilAddedDateList|length %}
```

Comment:
we are probably better off aligning these HTML changes with what Iswarya has done for her JIRAs
I think she take a null value into account
Please check with her